# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux24/gcc4.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-2.4.36/include/linux/compiler.h.orig	2008-02-17 13:29:55.000000000 +0100
+++ linux-2.4.36/include/linux/compiler.h	2008-02-17 13:30:45.000000000 +0100
@@ -42,4 +42,5 @@
 #endif
 
+#if 0
 #ifdef __KERNEL__
 #if __GNUC__ > 4 || __GNUC__ == 4 && __GNUC_MINOR__ >= 2
@@ -51,4 +52,5 @@
 #endif
 #endif
+#endif
 
 /* no checker support, so we unconditionally define this as (null) */
